N/ninja/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ninja"
```

Comment:
Small nitpick: the project is called "Ninja" according to https://ninja-build.org/
```suggestion
name = "Ninja"
```
Please also rename the directory to match this change